sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/EncryptionVersion.java

```diff
@@ -20,5 +20,12 @@ public enum EncryptionVersion {
      * Version 2 of the client side encryption protocol.
      * Uses AES/GCM/NoPadding
      */
-    V2
+    V2,
+
+    /**
+     * Version 2.1 of the client side encryption protocol. Use this version when configuring
+     * authenticatedRegionDataLength for encryption.
```

Comment:
I'm not sure if we need to add a code link in this docstring for this reference? It seems like it would be helpful if we can use it for linking.